mention-bot.js

```diff
@@ -285,12 +303,17 @@ function guessOwners(
       return owner !== creator;
     })
```

Comment:
since the getEligibleOwners function is just one line, you could add one more filter here.